.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "4.2.1"
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```

Comment:
copied over from the other one.  technically since we only use phantom (and not firefox or chrome) this is not required, but I would like to figure out why chromium/firefox don't work..